xtask/src/commands/changeset/mod.rs

```diff
@@ -0,0 +1,499 @@
+/// This module is generated:
+///   1. Only if you actually change the `matching_pull_request.graphql` query.
+///   1. By installing `graphql_client_cli`
+///
+///          cargo install graphql_client_cli
+///
+///   1. By running:
+///      1. A command that downloads the GitHub Schema.  It's large so we don't
+///         need to check it in.  Make sure to download it INTO the
+///         `src/commands/changeset/` directory.
+///
+///             wget https://docs.github.com/public/schema.docs.graphql`
+///
+///      2. Generate against this downloaded schema.  Run this from inside the
+///         `src/commands/changeset/` directory.
+///
+///             graphql-client generate \
+///               --schema-path ./schema.docs.graphql \
+///               --response-derives='Debug' \
+///               --custom-scalars-module='crate::commands::changeset::scalars' \
+///               ./matching_pull_request.graphql
+///
+mod matching_pull_request;
+mod scalars;
+
+use ::reqwest::Client;
+use anyhow::Result;
+use console::style;
+use dialoguer::{console::Term, theme::ColorfulTheme, Confirm, Editor, Input, Select};
+use git2;
+use itertools::Itertools;
+use matching_pull_request::matching_pull_request::Variables;
+use matching_pull_request::MatchingPullRequest;
+use memorable_wordlist;
+use serde::Serialize;
+use std::{fmt, fs, path::PathBuf, slice::Iter, str::FromStr};
+use structopt::StructOpt;
+use tinytemplate::{format_unescaped, TinyTemplate};
+use xtask::*;
```

Comment:
I slightly prefer not using wildcard imports as they make it harder read where a given item is from when reading code without rust-analyzer (such as in a PR review)

Reply:
That makes a lot of sense.  I had copied this from https://github.com/apollographql/router/blob/4e793352d01edb0d1fccf83c5c6d21143181d11d/xtask/src/commands/release.rs#L11 but I've changed it to not be that.  It's much nicer, for sure.

Fixed in 29c277d6